package.json

```diff
@@ -17,6 +17,7 @@
     "format": "prettier --write .",
     "protoc": "bash ./scripts/update_proto.sh",
     "test": "jest",
+    "test-all": "npm ci && npm run build --workspace=packages/utils && npm run build --workspace=packages/ledger && npm run test --workspaces",
```

Comment:
just sharing fyi, no need of action: I was curious about it - i.e. I asked my self if it's possible to build two workspaces in one command. Turns out it is.

> npm run build --workspace=packages/utils --workspace=packages/ledger

Reply:
Thanks. I think this way it's clearer that one is built before the other. But good to know.